### ISSUE TITLE
Inline preview does not resize pics, just crops them

### ISSUE THREAD

**Issue by Majki** [satanas/Turpial#326]
Just compiled libturpial 1.7 and turpial from git. Inline previews are now broken partially.

Example:

http://i.imgur.com/tIAZCLY.png

**Comment by satanas** [comment]
Hi, that's not an issue, it's the expected behavior. Turpial will crop the image to show you just a little preview (like the Twitter website). Then you can click the image to see it in full size

**Comment by Majki** [comment]
Oh my - previous feature was so much better imo and much more user friendly.

**Comment by satanas** [comment]
Yes I agree, but that feature was breaking out the whole layout of the app, so I had to remove it and just show a cropped version of the image :(

**Comment by khurshid-alam** [comment]
@satanas While I agree that the images should be cropped, the cropped size images are too small (in height). So basically I have to open each image just to understand what exactly it is about. Little more visible height doesn't harm.

Twitter (Web) shows full size image inline if the image size is relatively small. May be we can have similar mechanism here.

**Comment by satanas** [comment]
Ok, let's consider giving more size to the crop to improve the user experience

**Comment by Majki** [comment]
+1 khurshid-alam

Having to click each time for pic is annoying.

**Comment by jmcclelland** [comment]
I think it's reasonable to preview a smaller version of the image so we don't break the layout, however, I think the problem is the crop as opposed to resize. I suspect it will take more CPU to resize an image vs just cropping it, but I think the best solution would be to evenly resize it so you can get the whole image, just a smaller version of it.
